sdcm/tester.py

```diff
@@ -1368,6 +1370,9 @@ def run_stress_thread(self, stress_cmd, duration=None, stress_num=1, keyspace_nu
         elif stress_cmd.startswith('scylla-bench'):
             params['stop_test_on_failure'] = stop_test_on_failure
             return self.run_stress_thread_bench(**params)
+        elif stress_cmd.startswith('cassandra-harry') or 'cassandra-harry' in stress_cmd:
```

Comment:
```python
if 'cassandra-harry' in stress_cmd:
    ...
```
is enough here (it covers `.startswith()` case too)

Reply:
You are right. the startswith is unnecessary